API.md

```diff
@@ -0,0 +1,57 @@
+# Octofet API
+
+## `class Octofet`
+
+Uses: class Spi2Parallel
+
+Create an object of type `Octofet` to communicate with a particular [Octofet board](https://my.amperka.com/modules/octofet). The one board Octofet includes of 8 power switches.
+
+### `Octofet(uint8_t pinCS, uint8_t deviceCount = 1)`
+
+Constructs a new Octofet board object that uses the hardware SPI bus with arguments:
+
+- `pinCS`: pin Chip Select used for controls the shift-register latch. Any GPIO-pin can be used.
+- `deviceCount`: quantity of octofets boards connected in one chain: logical output pin headers of one module to the inputs of another. If the `deviceCount` argument is omitted, it used only one board Octofet.
+
+### `Octofet(uint8_t pinCS, uint8_t pinMOSI, uint8_t pinSCK, uint8_t deviceCount = 1)`
+
+Constructs a new Octofet board object that uses the software SPI bus with arguments:
+
+- `pinCS`: pin Chip Select used for controls the shift-register latch. Any GPIO-pin can be used.
+- `pinMOSI`: pin MOSI for sending data to the peripherals. Any GPIO-pin can be used.
+- `pinSCK`: pin SCK the clock pulses which synchronize data transmission generated by the master. Any GPIO-pin can be used.
+- `deviceCount`: quantity of octofets boards connected in one chain: logical output pin headers of one module to the inputs of another. If the `deviceCount` argument is omitted, it used only one board Octofet.
+
+Difference between hardware and software SPI is speed. Software SPI it will run much slower than hardware SPI, but hardware SPI works only on specific pins: `MOSI`, `MISO`(not used in Octofet) and `SCK`. Use software SPI only when hardware SPI pins are used by other devices.
+
+### `void begin()`
+
+Initializes the given interface, prepares the board for communication. All power switches are set to turn-off state.
+
+Important, call this method at least once, in your `setup()`.
+
+### `void digitalWrite(uint8_t channel, bool value, uint8_t device = 0)`
+
+Sets the state ("on" or "off") of one power switch. Function arguments:
+
+- `channel`: ordinal number power switch on device. Range is from `0` to `7`. Use option value is `ALL` to selected and sets state of all power switches.
```

Comment:
```suggestion
- `channel`: the power switch index. Ranges from `0` to `7`.
```

Do not complicate things. `ALL` is useless, let the user choose `digitalWrite8` instead.